srcpkgs/python3-calver/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-calver'
+pkgname=python3-calver
+version=2022.6.26
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3"
+short_desc="Setuptools extension for CalVer package versions"
+maintainer="Mohammed Anas <triallax@tutanota.com>"
+license="Apache-2.0"
+homepage="https://github.com/di/calver"
+changelog="https://github.com/di/calver/releases"
+distfiles="${PYPI_SITE}/c/calver/calver-${version}.tar.gz"
+checksum=e05493a3b17517ef1748fbe610da11f10485faa7c416b9d33fd4a52d74894f8b
+make_check=no # PyPI tarball has no tests.
+
+pre_patch() {
+	vsed -e "s/version=calver_version(True),/version=\"$version\"/" -i setup.py
+}
```

Comment:
Does the automated version detection fail to work as they expect?

Reply:
I did some tests and this package seems to be properly versioned without this patch. Am I missing something here?

Reply:
Indeed, I just checked myself and this line seems unnecessary. I'm sure I added this for a reason, but I can't remember what that was now...

Anyway, removed it.